node.js

```diff
@@ -1,43 +1,21 @@
-var semver = require('semver');
+var nonStandardPlugins = require('./non-standard-plugins');
 
 module.exports = function shopifyNodePreset(context, options) {
   options = options || {};
-  var version = options.version || process.version;
+  var version = options.version || true;
```

Comment:
Can we use `"current"` here, please?  Seems to [do the same thing](https://github.com/babel/babel-preset-env#targetsnode), and looks more explicit to me.